neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -48,7 +48,7 @@
     AlphaPropertyDTO,
     channel_type_to_operation,
     channel_value_type_to_operation,
-    deprecated_img_to_alpha_image,
+    deprecated_img_to_alpha_image, AlphaParameterDTO,
```

Comment:
Do you know how to configure PyCharm to do it?

Reply:
I can look form some plugins, but I'm not sure if it does it automatically :(
It can only reorder whole imports, not order of functions imported from particular module.

Reply:
Why is order important? If IDE can't do it for us automatically I would prefer not to worry about it.